### ISSUE TITLE
Update the Recommended/Supported information regarding GPU drivers

### ISSUE THREAD

**Issue by ghost** [openwall/john#5555]
- remove or update the “AMD driver name”;
  - I can think of at least two cases where the printed information is wrong. E.g., ROCm is reported as AMD Pro.
- remove or update the “Recommended” information;
  - in case of NVIDIA drivers, I bet that the 550 and similar drivers that magnum tested appear as recommended.

See #5554

**Comment by magnumripper** [comment]
Yeah that `[recommended]` for anything newer than some arbitrary, now ancient, version is misleading at best. I do not think we need it at all for nvidia - their driver has mostly been great all the time.

**Comment by solardiz** [comment]
Our driver list in `[List.OpenCL:Drivers]` is out of date, and not needed for NVIDIA. Should we remove this entire section, if we don't intend to update and maintain it for current AMD?

**Comment by ghost** [comment]
IMO, we can keep some documentation on it:

```
[List.OpenCL:Drivers]
#Driver ; Description         ; Recommendation
#AMD driver versions
1800, 5 ; 15.7                ; VG* R
1800, 8 ; 15.7.1              ; VGW R
#Acer, Dell (automagically installed)
3075, 13; 20.10.28            ; VGW R
#NVIDIA driver versions
535, 0  ;                     ; N*  T
550, 0  ;                     ; N*  T
560, 0  ;                     ; N*  R
#End
0, 0    ;                     ;
```

- Only known good AMD versions.
- Some NVIDIA notes.
- Being > 560 to be recommended? (untested).
- **[edited]** added my Windows AMD driver.
